assets/src/edit-story/components/library/panes/text/karma/textPane.karma.js

```diff
@@ -68,7 +68,7 @@ describe('CUJ: Creator can Add and Write Text: Consecutive text presets', () =>
       await waitFor(() => {
         expect(stripHTML(element.content)).toEqual(content);
         const preset = PRESETS.find(({ title }) => name === title);
-        expect(element.y).toEqual(preset.element.y);
```

Comment:
Maybe add a test for using a float to the font-size field as well (e.g. a unit test in addition to `'should select font size'`)?